ComposeSamples/ComposeGallery/src/main/java/com/microsoft/device/display/samples/composegallery/HomePage.kt

```diff
@@ -70,17 +75,43 @@ fun Home(viewModel: AppStateViewModel, windowInfoRep: WindowInfoRepository) {
 @Composable
 fun SetupUI(isDualMode: Boolean) {
```

Comment:
Seems `isDualMode` won't be needed anymore.

Reply:
But if you are adding the navigation button, it may still be needed. Same for `windowInfoRep`.